source/cloud/manage_volumes.md

```diff
@@ -6,7 +6,11 @@ volume to a running instance or detach a volume and attach it to another
 instance at any time. You can also create a snapshot from a volume, or
 delete it.
 
-## Create a volume
+## Terraform
+The terraform templates support volume provisioning for VMs.
+Since DATE, it can also mount and resize them automatically. See {ref}`volumes`
```

Comment:
I would remove since... and just say what it does now. We can add this into an email to the current users